qiskit/pulse/transforms/alignments.py

```diff
@@ -0,0 +1,479 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""A collection of passes to reallocate the timeslots of instructions according to context."""
```

Comment:
@taalexander I think you reviewed the original version of this file so I just very briefly skimmed it